client/tests/integration/smartcontracts/validator_with_admin/src/lib.rs

```diff
@@ -4,163 +4,24 @@
 #![no_std]
 #![allow(missing_docs, clippy::missing_errors_doc)]
```

Comment:
can you remove these lints now?

Reply:
I don't think so as the generated code is still exactly same. And it's mostly firing up because of the many `visit_...` functions whose names are self-descriptive.

Reply:
ok, but can you attach it to specific functions? this should not be a burden on the user